docs/02-authentication-and-access/enterprise-login.mdx

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 6
+sidebar_label: Enterprise login
+tags:
+    - security
+    - enterprise
+    - tutorial
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Enterprise login
+
+In this guide, we'll walk you through the process of setting up Enterprise Login with Security Assertion Markup Language (SAML) in Nblocks.
+This powerful feature allows you to seamlessly integrate your application with various Identity Providers (IdPs) supporting SAML.
+
+## Why Enterprise Login Matters
+If your app needs to scale for enterprise users. Security is non-negotiable, and centralized user management is a must, Enterprise login becomes a requirement.
+IT admins and managers can control who gets access to your app directly from their identity management system. 
```

Comment:
Suggestion: _If your app needs to scale for enterprise users, security is non-negotiable and centralized user management is a must. Consequently, enterprise login becomes a requirement. IT admins and managers can control who gets access to your app directly from their identity management system._